featurecontrol/featurecontrol.go

```diff
@@ -23,40 +23,40 @@ import (
 )
 
 const (
-	fcReceiverNameInMetrics  = "receiver-name-in-metrics"
-	fcClassicMatchersParsing = "classic-matchers-parsing"
-	fcUTF8MatchersParsing    = "utf8-matchers-parsing"
+	FeatureReceiverNameInMetrics = "receiver-name-in-metrics"
+	FeatureClassicMatchers       = "classic-matchers"
```

Comment:
Perhaps even something like this:

```suggestion
	FeatureClassicMode  = "classic-mode"
	FeatureUTF8Mode     = "utf8-mode"
```